src/runtime_src/core/tools/xbmgmt2/flash/flasher.h

```diff
@@ -79,7 +79,7 @@ class Flasher
 
```

Comment:
Copyright needs to be updated. A new line after 3 should be added to reflect AMD copyright. For example:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`